ruby-test-mode.el

```diff
@@ -178,20 +222,20 @@ element of each list in mapping, and replacing the match with the
 second element."
   (let ((target-filename nil))
     (while (and (not target-filename) mapping)
-      (let ((regexp-match (car (car mapping)))
+      (let ((regexp-match (eval (car (car mapping))))
```

Comment:
> Oh, I see. Eval is not necessarry when using backquotes as I mentioned above. Eval should be avoided if possible. :)

Cool, That is exactly what i want.
I think this case does not need eval too, and i know eval should avid.
 I just don't know what to do ... : (

Thanks.